### DISCUSSION TITLE
use VaultWarden offline

### DISCUSSION THREAD

**Discussion by FanFret** [dani-garcia/vaultwarden#4676] [Q&A] [answered]
Does anybody know, if it is possible to use the Bitwarden-App in an Offline-Mode? 
I have already installed the VW-server, connected it to the Bitwarden-App and everything works fine. 
Now I am trying to find a way to locally store a copy of the databse in case there is no network-connection and open it via Bitwarden. 
I am new here, so sorry, if I have overseen an answer already given or my wording is not quite correct. 
I hope you get the issue anyway. Best regards

**Comment by RealOrangeOne** [comment, marked as answer]
If you log in to the Bitwarden app, it keeps a cache of the DB locally - that way you can access your credentials without internet. When you're next online and open the app (and maybe when it's closed too), Bitwarden will sync any new data locally again. You don't need vaultwarden itself to be accessible "offline", so long as you're using a native app (mobile, desktop, or even the browser extension).

**Reply by FanFret** [reply]
Thank you for your reply. My experience is, that, as soon as i have logged of the app, and plugged the computer of the net, BW can not re-open. The error-announcement is "Failed to fetch". How can I solve this?

**Reply by RealOrangeOne** [reply]
It's going to fail to fetch, yes. But your credentials should all still be there and accessible.

**Reply by FanFret** [reply]
Where can I find them?

**Reply by FanFret** [reply]
Or - how can I open them? Thanks in advance!

**Reply by RealOrangeOne** [reply]
What do you mean? You just open the app and the credentials are there.

**Reply by FanFret** [reply]
I just found out, how it works: Not logg-off from the app, but simpl end it. This way I can re-access the DB. Thank you!

**Reply by RealOrangeOne** [reply]
Oh, yes. If you log out you'll delete the local cache. Closing the app or explicitly "locking" it will achieve what you want.

**Comment by Joackk** [comment]
Hey, do you get very stuck and slow using browser plug-ins while offline?
